src/univariate/discrete/bernoulli.jl

```diff
@@ -80,8 +80,8 @@ end
 
 #### Evaluation
 
-pdf(d::Bernoulli, x::Bool) = x ? succprob(d) : failprob(d)
-pdf(d::Bernoulli, x::Int) = x == 0 ? failprob(d) :
+pmf(d::Bernoulli, x::Bool) = x ? succprob(d) : failprob(d)
+pmf(d::Bernoulli, x::Int) = x == 0 ? failprob(d) :
```

Comment:
no need to add two methods nor to hard-code `Int`:
`Bool <: Integer` and `true == T(1)` for any `T <: Integer`

Reply:
Again, this isn't my code - all I've done is searched and replaced pdf with pmf for discrete distributions. I'm sure there are lots of problems with the existing code, but I felt that I was already being ambitious enough!